share/completions/rustc.fish

```diff
@@ -49,13 +49,13 @@ for line in $rust_docs
 end
 
 set -l rust_docs (rustc -W help  \
```

Comment:
I feel like there should be some means with just string replace and the pcre2 syntax if I better understood regular expressions I could do lines 52-55 with one string replace... I can't nail it.